python/plugins/processing/ProcessingPlugin.py

```diff
@@ -75,6 +75,7 @@ def handleFileDrop(self, file):
         if not file.lower().endswith('.model3'):
             return False
         self.runAlg(file)
+        return True
```

Comment:
Should that be `return self.runAlg(file)`?

Reply:
@m-kuhn as stated in the PR description, this part come from a not yet PR by @nyalldawson https://github.com/qgis/QGIS/pull/7387 I preferred to anticipate it and leave the PR in WIP to have a code to test and a place where to discuss. I'll rebase as soon as nyall's PR will be merged.

Reply:
Alright, I was just stumbling over this so I though I leave a note for you, feel free to review this whenever you think it's appropriate :)

Reply:
@m-kuhn Fixed in #7387